plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -268,7 +268,7 @@ func ClusterRoles() []rbac.ClusterRole {
 				// Used to build serviceLister
 				rbac.NewRule("list", "watch").Groups(legacyGroup).Resources("services", "endpoints").RuleOrDie(),
 				rbac.NewRule("get").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
-
+				rbac.NewRule("patch").Groups(legacyGroup).Resources("nodes/status").RuleOrDie(),
```

Comment:
I am open to suggestions. The KubeProxyVersion lives in an awkward place that requires an update. My only other idea was to do the reverse and have the kubelet contact the kube-proxy which I thought was a bit worse.